src/jvm/main/org/jetbrains/kotlinx/lincheck/trace/TracePoint.kt

```diff
@@ -239,6 +240,10 @@ internal class MethodCallTracePoint(
     var thrownException: Throwable? = null
     var parameters: List<String>? = null
     var parameterTypes: List<String>? = null
+    
+    var isSuspendMethodCall = false 
+        private set
```

Comment:
Can we compute it in the constructor?

Reply:
Wether or not a function can be marked as suspend currently depends on it's parameters. These parameters are initialized after construction by `ManagedStrategy` calling `MethodCallTracePoint.initializeParameters`.

Reply:
I agree we should initialize all properties of all trace points in their constructors (and purge all `lateinit` properties), but it is probably worth doing in a separate PR.

Reply:
#626